### PR TITLE
Added okta docs to its own branch

### PR DESCRIPTION
**Pull request by gabecase**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/staging.horse/build-c324ad0e39/docs/rackspace-federation/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/staging.horse/build-b458113829/docs/rackspace-federation/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/staging.horse/build-b2a4bc59a6/docs/rackspace-federation/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/staging.horse/build-629e8d5bc8/docs/rackspace-federation/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/staging.horse/build-7ff3e7b075/docs/rackspace-federation/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/staging.horse/build-4d77e12c6a/docs/rackspace-federation/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/staging.horse/build-f311eeaacb/docs/rackspace-federation/). :bow:

File: doc/config-3p-saml/config-okta/okta-setup.rst [hunk 1]
```diff
@@ -42,11 +37,13 @@ SAML assertions.
      - https://login.rackspace.com/federate/acs
 
 
+Instructions on setting up SAML applications in Okta can be found here:
```
Comment (gabecase) [inline doc/config-3p-saml/config-okta/okta-setup.rst]: Moved this under the the information on what to put into the SAML configuration as a reference for users who need more help with okta SAML configurations.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/staging.horse/build-e8257d559d/docs/rackspace-federation/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/staging.horse/build-dcc1c7a3ff/docs/rackspace-federation/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/staging.horse/build-d423322a61/docs/rackspace-federation/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/staging.horse/build-4d1d45a41d/docs/rackspace-federation/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/staging.horse/build-8943a618a2/docs/rackspace-federation/)

**Pull request merged**